### DISCUSSION TITLE
"New theoretical model claims to solve mystery of early massive galaxies"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#13] [Astrophysics and Physics]
[New theoretical model claims to solve mystery of early massive galaxies](https://phys.org/news/2023-05-theoretical-mystery-early-massive-galaxies.html)
"Astrophysicists from the Hebrew University of Jerusalem published a new theoretical model that solves the mystery of the formation of early massive galaxies in the universe, in Monthly Notices of the Royal Astronomical Society."
"Prior to the enrichment of the gas by heavy elements produced in stars, the researchers suggest star-forming clouds in the dense [early universe](https://phys.org/tags/early+universe/) had a density above a threshold that allowed rapid collapse of the gas into stars"

**Comment by budrap00** [comment]
Of course, of course and this blatant post-diction is already being declared a predictive success:

> "The predictions of this model will be tested using the accumulating new observations from the Web Space Telescope, where it seems that some of these predictions are already confirmed."

There is a need for a new word that combines the attributes clueless and shameless for people who engage in this sort of behavior.

**Reply by ftessier** [reply]
`shueless`?

_(suggestion by ChatGPT)_
